analysis/5/dsp_analysis.ipynb

```diff
@@ -0,0 +1,671 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import seaborn as sns\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline \n",
+    "sns.set(color_codes=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(r'/Users/priya/Downloads/crude_suicide_rates-2.csv')"
```

Comment:
- Using a relative path is preferable to an absolute path, so the script could run on another computer.

Since we already have a class to read the dataset, maybe you could modify your code to:

```
import os
import sys
sys.path.append(os.path.abspath('..'))
from common_lib.data_reader import SuicideDataReader, SuicideProcessedData
data_reader = SuicideDataReader()
data = data_reader.read_data(SuicideProcessedData.SUICIDE_RATES, tidy=True)
```

- is crude_suicide_rates-2 different with current dataset (crude_suicide_rates)?

Reply:
I tried using the absolute path.. keep running into an error.. it says - module "common_lib" not found.

yes, crude_suicide_rates-2 is same as crude_suicide_rates

Reply:
One thing to check is that if you find  "common_lib" under /analysis folder.

If it is not found, this might be that your local repo is not updated. You may try doing
`git pull  https://github.com/mdsi-dsp-at2/eTe-analysis`